### ISSUE TITLE
Chrome, IE Edge Twitching of the first column with categories

### ISSUE THREAD

**Issue by ViktoriaNeva** [levelupik/levelupik#33]
Steps To Reproduce:
1. Click on the button for creating category

Actual Result:
Twitching
It has never been before. I think it started after the last release 0.2.1
![2020-05-29_18-25-08](https://user-images.githubusercontent.com/5111608/83279901-7d95c380-a1de-11ea-920b-8397c52c456e.png)
![2020-05-29_18-25-19](https://user-images.githubusercontent.com/5111608/83279913-81c1e100-a1de-11ea-9328-07e4a2c7ee66.png)

Expected Result:
No twitching

**Comment by levelupik** [comment]
This block has not been changed on this version, so maybe you see it just because of some other changes in your categories(Lorem Ipsum for example).

After adding a new category it will have to get status `focus` for quick inserting text reasons.
If a new category-creating on the not visible box area, the category box will jump to it.

So, for example, you have short category names and some long ones.
Now, if you are will activate some short name category and scrolled by horizontal to the right as far as you can - you will not see other categories names but just a longer one.
If you click `add` - you will see that visible box jumped to the left, for catching new category creation.

------
Can you describe more detailed steps to reproduce, please?
Is it on the Windows operation system?

**Comment by ViktoriaNeva** [comment]
>>This block has not been changed on this version, so maybe you see it just because of some >>other changes in your categories(Lorem Ipsum for example).
Yes. This is because of long title of the category (I will add it to the STR)

>>Is it on the Windows operation system?
No. It's Mac. But I believe there is no difference between OSs in this question. 

So, Is it as designed? 
To tell the truth, I don't like that icons of the categories do not shows anymore after "jumping" and titles too close to the left side of the form.
